charts/rollout-operator/values.yaml

```diff
@@ -12,6 +12,7 @@ revisionHistoryLimit: 10
 
 image:
   repository: grafana/rollout-operator
+  registry: ""
```

Comment:
Why is this needed? Couldn't the registry be added to `repository` instead? (eg. `us.gcr.io/foo/blah/rollout-operator`)